### ISSUE TITLE
Weapon Balance Compendium 2

### ISSUE THREAD

**Issue by Gregory-Y** [deferred-impact/battledudes-issue-tracker#73]
Basically this compilation of suggestions is meant to make assault rifles more distinct from SMGs. The removal of falloff is intended to make assault rifles more effective at long range and less effective at close range. On the other hand, SMGs and shotguns will be made more potent at close range while at long range they are much more difficult to use. Other weapons will also see minor changes: sniper rifles have received minor buffs and pistols have been made a bit less damaging but falloff for them has been reduced.

So yeah. Don't be alarmed by the sharp decreases in damage for all the assault rifles, because falloff has been largely removed from them and they will be more consistent across all ranges. This will make assault rifles require more skill to use (better aim), and it will establish assault rifles as a more distinct weapon class which still functions as a jack-of-all trades when it comes to their role within the meta.

Remove falloff from most assault rifles. Keep falloff for most SMGs and shotguns (with adjustments).

Nerf M4 to 9 damage. Increase accuracy to 1.05.
Nerf AK47 to 10.5 damage. Increase accuracy to 0.9. Keep falloff for this one, but make it lower at like 0.15.
Nerf M249 and SCAR to 11 damage. Buff SCAR bullet speed to 87-88 and M249 bullet speed to 83. Increase SCAR accuracy to 0.7.
Nerf Falcon to 13.5 damage. Buff bullet speed to 87-88. Increase accuracy to 0.9. Keep falloff for this one, reduce to 0.15.
Nerf FAMAS to 12.5 damage. (or maybe make it 13.5 but make falloff like 0.15)

Make all sniper rifles have first shot accuracy with a cooldown of 0.6 seconds during which you must not fire the weapon. Display something that indicates FSA recharge. Fix desync for Heavy Sniper, if not the others.
Nerf heavy sniper fire rate to like 600ms (so it essentially has perfecc accuracy).
Make light sniper reload 1 bullet at a time like the pump sg, as outlined in #13 .
Nerf M1 accuracy to like 0.65 but keep the fire rate. (Consider removing M1 because there doesn't seem to be a good way to balance it with the light sniper and falcon.)

Nerf 9mm smg to 10 (or 10.5, if express and kevlar gets removed, see perk balance compendium) damage. Reduce falloff to 0.25. Reduce accuracy to 0.5.
Nerf micro smg to 12 damage. Reduce falloff to 0.5.
Nerf P50 to 6.5 damage. Remove falloff. (Ideally, remove P50. Just put the poor thing out of its misery.)
Nerf Vector to 9.5 damage. Reduce falloff to 0.6.

Make Sawed Off shotgun a secondary (if Sarunas really can't get it to just not take extended magazines). Reduce sawed off fire rate to around 380ms. Buff its damage to 11. Possibly buff the spread to be more consistent.
Buff Pump shotgun damage to 10.5 and increase pellet count to 6. Increase falloff to 0.8.
Auto shotgun is fine I think.

Keep 1912 the same.
Reduce Deagle falloff to 0.35.
Reduce CZ75 damage to 7.5. Reduce falloff to 0.45.
Remove falloff for Revolver, it doesn't do anything anyway. Revolver should remain a 2 shot for consistency's sake. Fix desync for revolver.
Nerf 9mm burst to 11 damage but buff accuracy to 0.4.
Crossbow should be a guaranteed 2 shot. It has no falloff and 30x2=60. As such, xbow does not need any changes but it should get the same treatment as flint when it comes to desync.
Buff flint to have perfect accuracy, not that that changes much. Also make flint unlockable at level 10.

Add a cooldown for flashbangs and decrease their capacity to be the same as that of sticky grenades to reduce flash spamming.
Implement mine changes outlined in #18 .
Also make frags interceptable by trophies, it really does not take much effort to destroy a trophy system especially since most players have the armed perk in gamemodes where trophy really matters like payload.

Implement more shield fixes (#58  and #50 ).

**Comment by DisposableKatyusha** [comment]
1912 is a literal water gun, no need for nerf

**Comment by snipergamingop** [comment]
I agree a lot with ARs and shotgun reworks, not so sure about SMGs, might be better than my changes. Sidearms reworks are nice tho

**Comment by Gregory-Y** [comment]
I suppose 1912 could remain at 13 damage.

**Comment by deferred-impact** [comment]
We need to rework this using some actual math. (Sorry i forgot to tell you, i asked the dev for the accuracy stat formula a couple of days ago, and he did provide it). Instead of picking a random number that simply "feels right", i'd love for these nerfs to be more rigorous. Especially the damage and accuracy changes

**Comment by Gregory-Y** [comment]
Yeah I just kinda used the vector's accuracy as a reference point when I originally did this lmao. Would be nice if you could share the accuracy formula in discord.

**Comment by JacKDot2** [comment]
Pump buff seems good

**Comment by deferred-impact** [comment]
> Pump buff seems good

Hmm i wonder why
